<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1634</width>
    <height>1158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="howmany">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>How many?</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="people_group_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>1</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Regular</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="people_group_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>2</string>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Regular</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="people_group_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>3</string>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Regular</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="people_group_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>4</string>
    </property>
    <widget class="QLineEdit" name="p_start_lbl_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Regular</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>1631</width>
      <height>981</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="subway_map.qrc">:/subwaymap/subwaymap.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="people_number_c_box">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="minimumContentsLength">
     <number>1</number>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>2명</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3명</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4명</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="home">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>20</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>1훈솜사탕 R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="subway_map.qrc">:/subwaymap/snow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="topbgcolor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1651</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>860</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="topbgcolor_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1000</y>
      <width>1661</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="run_button2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>topbgcolor</zorder>
   <zorder>topbgcolor_2</zorder>
   <zorder>howmany</zorder>
   <zorder>people_group_1</zorder>
   <zorder>people_group_2</zorder>
   <zorder>people_group_3</zorder>
   <zorder>people_group_4</zorder>
   <zorder>label_picture</zorder>
   <zorder>people_number_c_box</zorder>
   <zorder>home</zorder>
   <zorder>label</zorder>
   <zorder>run_button2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="subway_map.qrc"/>
 </resources>
 <connections/>
</ui>
